cmd/juju/model/exportbundle.go

```diff
@@ -38,12 +39,17 @@ Exports the current model configuration as a reusable bundle.
 If --filename is not used, the configuration is printed to stdout.
  --filename specifies an output file.
 
+If --export-with-series is used, the configuration will include series data
+ alongside bases. This should be used as a compatibility option if you wish your
+ bundle to be compatible with versions of juju before bases were introduced
+
 `
 
 const exportBundleHelpExamples = `
     juju export-bundle
     juju export-bundle --filename mymodel.yaml
     juju export-bundle --include-charm-defaults
+	juju export-bundle --export-with-series
```

Comment:
is the indentation messed up?

Reply:
Yep. the text editor on my laptop doesn't show the difference between tabs and spaces, so I've made this mistake a few times last few days